packages/smooth_app/lib/pages/product/common/product_list_helper.dart

```diff
@@ -0,0 +1,84 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:smooth_app/data_models/product_list.dart';
+import 'package:smooth_app/database/dao_product.dart';
+import 'package:smooth_app/database/dao_product_list.dart';
+import 'package:smooth_app/database/local_database.dart';
+import 'package:smooth_app/pages/product/common/product_refresher.dart';
+
+// ignore: avoid_classes_with_only_static_members
+class ProductListHelper {
```

Comment:
I would avoid `ignore` and "static" classes, especially given that in that case we could imagine something like that:
```suggestion
class ProductListHelper {
  final ProductList list;
  final LocalDatabase localDatabase;
  const ProductListHelper(this.list, this.localDatabase);
  String exportBarcodesToString() { ... }
  Future<void> importListFromFile(File file) { ... }
```